README.md

```diff
@@ -12,6 +12,10 @@ Simplest example:
 
 [![asciicast](https://asciinema.org/a/388121.svg)](https://asciinema.org/a/388121)
 
+##### About `composer`
+
+> All images currently ship out with Composer 2.x as the default version. If for some reason you still need to use Composer 1.x and cannot make the upgrade, we still ship a `composer1` in the images PATH as well, which is latest Composer 1.x version. Feel free to use it while you prepare to move to 2.0.
```

Comment:
maybe worth mention we still have the `composer2` symlink for BC.

Reply:
I don't see the point in mentioning it, since no one but who already has it will keep using it... so it is not a matter of concern for the general public.